app/src/controllers/testPlan/constants.ts

```diff
@@ -24,8 +24,8 @@ export const ACTIVE_TEST_PLAN_NAMESPACE = 'activeTestPlan' as const;
 export const TEST_PLAN_FOLDERS_NAMESPACE = 'testPlanFolders' as const;
 export const TEST_PLAN_TEST_CASES_NAMESPACE = 'testPlanTestCases' as const;
 export const defaultQueryParams = {
-  limit: 1000,
-  sortBy: 'createdDate,desc',
+  limit: 20,
+  offset: 0,
```

Comment:
Can you clarify why we are changing the limit and also removing the `sortBy`? cc @yelyzavetakhokhlova

Reply:
If you add new item will it appear on top of the list without the `sortBy` param?

Reply:
@ViktorSoroka07  No.
It will appear at the end of the list. But this logic we had before. You can check it on our env that does not have my changes.